api.planx.uk/modules/webhooks/service/analyzeSessions/operations.ts

```diff
@@ -20,6 +21,8 @@ const ALLOW_LIST = [
   "application.information.sensitive",
   "application.type",
   "drawBoundary.action",
+  "_feedback",
+  "_feedback.feedbackScore",
```

Comment:
Not entirely sure whether `_feedback.feedbackScore` is needed here, and if it is, whether it is in the right format... The `sql` file does not have `_feedback.feedbackScore` in it directly, but seemed to work as it is!

e.g. this is from local hasura, analytics table:
![Screenshot 2024-11-14 at 17 06 11](https://github.com/user-attachments/assets/d4a6b1d0-1204-4d1f-a2d1-698c39ddd30f)

Reply:
Sorry for confusion here! Here's what I'd expect:
- _only_ `"_feedback"` should be included in this list. Dot-separated variables only work when it's the "whole" property name and doesn't require destructuring a value, which `"_feedback.feedbackScore"` would
- This means that `feedback` column in the raw analytics table will look exactly like you've posted above - a jsonb object ! 
- then in the SQL to generate the view you should be able to do something like `((ls.allow_list_answers -> '_feedback' -> 'feedbackScore'::int))::int AS feedback_score` so that what we're exposing to Metabase is the feedback score as an integer (Metabase can't process jsonb! And free-text `userComments` are nice to track in raw table but aren't very chart-able)

Does this make sense?

Reply:
Thanks Jess, the `int` version you put above (and variations I have tried) caused syntax errors `"message": "invalid input syntax for type integer: \"feedbackScore\"",` - I assume the reason it shouldn't be text is because Metabase can make nice charts with numbers and can't convert from string numbers to actual numbers?

Have added a column for userComment in the table, and I assume that the feedback column (displaying jsonb) is completely redundant then, given that we now have separate columns for `feedback_score` and `user_comment`. Figuring out a way to drop that column from the view if so!

Reply:
<img width="677" alt="Screenshot 2024-11-18 at 21 02 44" src="https://github.com/user-attachments/assets/a12debba-c223-476d-bee2-979698c34b33">